snowpackjs/astro
# Fix/issue #385

## Changes

- Apply favicon to the docs template
- Fix Sidebar category headers aren't clickable
- Improve Docs example documentation

## Testing

In `\examples\docs`:
- run `npm install`
- run `npm run start`

The project now should have the following navigation items `Introduction`, `Getting Started`, and `Example`.
The Introduction item should be a link to the root page.
The favicon should now be displayed (on browsers that support SVG images).

- [ ] Tests updated where necessary

## Docs

- [x] Docs / READMEs updated
- [ ] Code comments added where helpful
